src/components/app.jsx

```diff
@@ -1,5 +1,14 @@
+/* eslint-disable react/prefer-stateless-function */
 import React from 'react';
 
-const App = () => (<h1>Hello world!!!</h1>);
+import ItemTitle from './ItemTitle.jsx';
```

Comment:
We don't want to import this here on the master version. I suggest keeping it on your branch while you're testing but we should keep the master branch clear of code we know we are going to remove.